timesketch/api/v1/resources/searchtemplate.py

```diff
@@ -100,21 +100,19 @@ def get(self, searchtemplate_id):
         Returns:
             Search template in JSON (instance of flask.wrappers.Response)
         """
-        form = request.json
-        parameters = {}
-
-        if form:
-            parameters = form.get("parameters", {})
-
+        form = request.json or {}
         searchtemplate = SearchTemplate.query.get(searchtemplate_id)
         if not searchtemplate:
             abort(HTTP_STATUS_CODE_NOT_FOUND, "Search template was not found")
 
         try:
             template = jinja2.Template(searchtemplate.query_string)
-            parsed_query_string = template.render(parameters)
+            parsed_query_string = template.render(form)
+            parsed_query_string = parsed_query_string.replace("\\/", "\\/")
```

Comment:
Is that something that might end up required in multiple places? If so maybe we should add a util function?
Also I would suggest we add some tests to unit tests that those replacements provide what they are suppose do.

Reply:
I agree, let's do it now rather than later. I refactored this into a utility function and added tests.